test/e2e/apimachinery/aggregator.go

```diff
@@ -314,7 +314,23 @@ func TestSampleAPIServer(f *framework.Framework, image string) {
 	// is setting up a new namespace, we are just using that.
 	err = framework.WaitForDeploymentComplete(client, deployment)
 
-	// We seem to need to do additional waiting until the extension api service is actually up.
+	// Waiting until the extension api service is actually up.
+	err = wait.Poll(5*time.Second, 5*time.Minute, func() (bool, error) {
```

Comment:
Why so long interval? Why not 100*time.Millisecond, 30*time.Second as below?

Reply:
5 time.Minute is a max wat interval. If the api service is up at second 5, this `wait.Poll` method will return early.
30 seconds may be too short.